docs/management/admin/trusted-apps.asciidoc

```diff
@@ -14,6 +14,8 @@ Trusted applications create blindspots for {elastic-defend}, because the applica
 
 Trusted applications might still generate alerts in some cases, such as if the application's process events indicate malicious behavior. To reduce false positive alerts, add an <<endpoint-rule-exceptions,Endpoint alert exception>>, which prevents {elastic-defend} from generating alerts. To compare trusted applications with other endpoint artifacts, refer to <<endpoint-artifacts>>.
 
+Additionally, trusted applications still generate some `create` and `terminate` process events for visualizations and other internal use by the {stack}. To prevent process events from being written to {es}, use an <<event-filters,event filter>> to filter out the specific events that you don't want stored in {es}.
```

Comment:
oops, one of those @gabriellandau 's was supposed to be @nicholasberlin

Reply:
> To prevent process events from being written to {es}, use an <<event-filters,event filter>> to filter out the specific events that you don't want stored in {es}.

This part is correct.  Every single `Events` event must pass through `Policy::IsEventFilterListed` to get to `Comms::Append`, and the former has no special treatment for process events.  Process events can be dropped by event filters, like any other event type.  Event filtering does not apply to non-`Events` documents like heartbeats, alerts, metadata, response actions, and policy results.

> Additionally, trusted applications still generate some `create` and `terminate` process events for visualizations and other internal use by the {stack}.

Upon matching a TrustedApp condition, any event with `EventLib::EVENT_TYPE_PROCESS == event->Type()` will have `process.Ext.trusted` set to `true`, but will still be passed down the pipeline to the `Comms` plugin.  If fork/exec events are `EventLib::EVENT_TYPE_PROCESS`, then this would apply to them as well.  Per `EventUpdater::UpdateEvent`, Fork and Exec events appear to be `EventLib::EVENT_TYPE_PROCESS`, so it looks like they will still be emitted for TrustedApps.

@nicholasberlin can you confirm?

Reply:
We finished the discussion elsewhere and concluded that all process event types are currently emitted for TrustedApps.  Suggested change below.

```suggestion
Additionally, trusted applications still generate some process events for visualizations and other internal use by the {stack}. To prevent process events from being written to {es}, use an <<event-filters,event filter>> to filter out the specific events that you don't want stored in {es}.
```

Reply:
Sounds good, thanks for the wider context and review, @gabriellandau @ferullo and all!